Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,30 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            for (int i = 0; i < 2; i++)
+            {
+                Console.WriteLine("Enter person " + (i+1) + " first name:");
+                string fName = Console.ReadLine();
+
+                Console.WriteLine("Enter " + fName + "'s last name:");
+                string lName = Console.ReadLine();
+
+                Console.WriteLine("Enter " + fName + " " + lName + "'s age:");
+                int age = Convert.ToInt32(Console.ReadLine());
+
+                Console.WriteLine("Enter " + fName + " " + lName + "'s weight in kg:");
+                double weight = Convert.ToDouble(Console.ReadLine());
+
+                Console.WriteLine("Enter " + fName + " " + lName + "'s height in cm:");
+                double height = Convert.ToDouble(Console.ReadLine());
```

Comment:
Those can be float, too.
You can also use float.Parse() directly here.

Reply:
If I am understanding your comment right. You are suggesting that I write it like this?

float weight = float.Parse(Console.ReadLine());

Reply:
Yes exactly! 👍